src/app/Caches/Create/Review.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useState, useEffect } from 'react';
+import {
+    Button,
+    Checkbox,
+    Divider,
+    Form,
+    FormGroup,
+    NumberInput,
+    Radio,
+    Select,
+    SelectOption,
+    SelectVariant,
+    Text,
+    TextContent,
+    TextVariants,
+    TextArea
+} from '@patternfly/react-core';
+import { CacheType, EncodingType } from "@services/infinispanRefData";
+import { useTranslation } from 'react-i18next';
+import { MoreInfoTooltip } from '@app/Common/MoreInfoTooltip';
+import { ReviewOptions } from "@app/Caches/Create/CreateCacheWizard";
+
+const Review = (props:
+    {
+        reviewOptions: ReviewOptions;
+        reviewOptionsModifier: (ReviewOptions) => void
+        createCacheConfig: () => any
+    }) => {
+
+    const { t } = useTranslation();
+
+    // State for the form
+    // Passed to the parent component
+    const [createCacheCode, setCreateCacheCode] = useState(props.reviewOptions.createCacheCode);
+    const [downloadType, setDownloadType] = useState<"json" | "yaml" | "xml">(props.reviewOptions.downloadType);
+
+    const [isOpenDownloadType, setIsOpenDownloadType] = useState(false);
+
+    const data = props.createCacheConfig();
+
+    useEffect(() => {
+        // Update the form when the state changes
+        props.reviewOptionsModifier({
+            createCacheCode: createCacheCode,
+            downloadType: downloadType
+        });
+    }, [createCacheCode, downloadType]);
+
+    /*TODO: Create similar config for Replicated cache */
+    // Distributed Cache config with variables
+    const distributedCache = {
```

Comment:
@karesti If it's the right way to create cache config?

Reply:
no. I will add a commit